common/lib/xmodule/xmodule/course_module.py

```diff
@@ -740,6 +740,18 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    allow_proctoring_opt_out = Boolean(
+        display_name=_("Allow Opting Out of Proctored Exams"),
+        help=_(
+            "Enter true or false. If this value is true, the learner is given the option to choose "
```

Comment:
@andy-armstrong Do you mind tweaking this help text a bit? 
"If this value is true, learners can choose to take proctored exams without proctoring. If this value is false, all learners must take the exam with proctoring. This setting only applies if proctored exams are enabled for the course."